code/modules/admin/outfit_editor.dm

```diff
@@ -179,6 +179,10 @@
 			choose_any_item(slot)
 		if("back")
 			options = typesof(/obj/item/storage/backpack)
+			for(var/obj/item/mod/control/pre_equipped/potential_mod as anything in typesof(/obj/item/mod/control/pre_equipped))
+				if(!(potential_mod.slot_flags == ITEM_SLOT_BACK))
```

Comment:
This shouldn't work at all, it should runtime error because you're checking paths, you'll have to check initial() or new it.

Reply:
fixed and double-tested that it works